app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationsFragment.kt

```diff
@@ -453,7 +453,7 @@ class NotificationsFragment :
         // Save the ID of the first notification visible in the list
         val position = layoutManager.findFirstVisibleItemPosition()
         if (position >= 0) {
-            adapter.snapshot()[position]?.id?.let { id ->
+            adapter.snapshot().getOrNull(position)?.id?.let { id ->
```

Comment:
We have a few "crashes but I don't know what leads to it" now (I fixed two).

Maybe this would be a case for an error log - being shown in the new application event log.

Reply:
This is the sort of thing that could be covered, yes.